tests/unit/package_managers/test_gomod.py

```diff
@@ -262,7 +271,9 @@ def test_resolve_gomod_vendor_dependencies(
         proc_mock(
             "go list -e -m",
             returncode=0,
-            stdout="github.com/cachito-testing/gomod-pandemonium",
+            stdout=get_mocked_data(data_dir, "non-vendored/go_list_modules.json").replace(
```

Comment:
To use `Template` here, we'd need to either subclass it so it would understand curly brackets as placeholders, or change the script that generates all the data for unit tests to use `$` as a template delimiter. I think replace is a easier solution here. Are you ok if we just leave it this way?